src/overlays/actors/ovl_En_And/z_en_and.h

```diff
@@ -2,16 +2,37 @@
 #define Z_EN_AND_H
 
 #include "global.h"
+#include "objects/object_and/object_and.h"
 
 struct EnAnd;
 
 typedef void (*EnAndActionFunc)(struct EnAnd*, PlayState*);
 
 typedef struct EnAnd {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x44];
+    /* 0x144 */ SkelAnime skelAnime;
     /* 0x188 */ EnAndActionFunc actionFunc;
-    /* 0x18C */ char unk_18C[0x188];
+    /* 0x18C */ u8 action;
+    /* 0x18D */ char pad18D[0x7];
+    /* 0x194 */ Vec3f headComputedPos;
+    /* 0x1A0 */ Vec3f torsoComputedPos;
+    /* 0x1AC */ Vec3s headComputedRot;
+    /* 0x1B2 */ Vec3s torsoComputedRot;
```

Comment:
```suggestion
    /* 0x194 */ Vec3f headTranslate;
    /* 0x1A0 */ Vec3f torsoTranslate;
    /* 0x1AC */ Vec3s headRotate;
    /* 0x1B2 */ Vec3s torsoRotate;
```
Looks like `EnSGoro` has similar variables and these are what they are named, so would be better to keep them consistent.

Reply:
Ok. I found the naming from `EnNb`, so it appears there is a conflict.

Reply:
Ah, Hmm..., in that case I say leave it as is and we can standardize them later.